### ISSUE TITLE
Slightly misleading status message in instructorCourseEnrollSave after enrollment

### ISSUE THREAD

**Issue by thyageshm** [TEAMMATES/repo#2079]
Currently after the instructor "enrolls" student(s) he receives success information and is given the option to "modify  values and REDO the enrollment". 

Possibly we could reword this as it might imply that the current enrollment is undone once the instructor clicks on the link

![studentenrollsuccessstatus](https://cloud.githubusercontent.com/assets/5160137/3099893/9680e21e-e608-11e3-94e4-cca729f49eff.jpg)

**Comment by taniach** [comment]
May I fix this?

**Comment by thyageshm** [comment]
Go ahead and create a pull request once you are done
